docs/_docs/themes.md

```diff
@@ -50,8 +50,16 @@ To locate a theme's files on your computer:
    ```sh
    # On MacOS
    open $(bundle show minima)
+
    # On Windows
-   explorer /usr/local/lib/ruby/gems/2.3.0/gems/minima-2.1.0
+   # First get the gem's installation path:
+   #
+   #   bundle show minima
+   #   => C:/Ruby24-x64/lib/ruby/gems/2.4.0/gems/minima-2.1.0
```

Comment:
What about this output?

Reply:
```
explorer C:/Ruby24-x64/lib/ruby/gems/2.4.0/gems/minima-2.1.0
```
wont work because the native file separator on Windows is `\`